src/patch-events.js

```diff
@@ -275,6 +275,10 @@ export function addEventListener(type, fn, optionsOrCapture) {
           return;
         }
       }
+      if(fn.handleEvent) {
```

Comment:
The tests are currently failing on master: https://github.com/webcomponents/shadydom/issues/80